# gnss
origin: [37.451069, 126.655925, 53.377] 
# origin: [37.4480025, 126.6534866, 33.741] # 분수대 필테 중앙
# origin: [35.06965949, 128.57880311, 50.921] # 창원

# 수정 필요
# autonomous_goal: [35.0693232, 128.5788879, 49.451] # 0 8/7일에 땄던 좌표 35.0693838, 128.5789538

# latitude, longitude, altitude


#For using dummy gps
waypoints: [
 [37.451069, 126.655925, 53.377], # WP4
 [37.45111, 126.655848, 51.761], # WP1
 [37.451094, 126.655874, 51.641], # WP2
 [37.450982, 126.655998, 52.006], # WP3
 [37.451069, 126.655925, 53.377], # WP4
]

goal_range: 1 #[m] # float이면 터짐 수정 필요

# Thruster
base_thruster: 1550
kp_thruster : 2.0
kd_thruster : 0.3
thrust_range: [1350, 1650]
delta_dot_max: 4 # [rad/s] 2.09

controlangle : 70
angle_num: 20
yaw_range: 60
range: 5
margin: 0.1

# image sensing
video_num: 1
target_shape: 'Cross'
color_to_detect: 'blue'

######################################
#     지구상의 모든상황의 변수를 통제       #
######################################
front_distance_1: 2.6
yaw_target1: 160
front_distance_2: 1.5
yaw_target2: 100
yaw_target3: 290
min_safe_distance: 1.0
min_angle_diff: 15

######################################
#             PredockCal             #
######################################

front_angle_range: 
- [175, 180]
- [-185, -180]
right_angle_range: 
  - [85, 95]  # ✅ 이렇게 하면 `[[85, 95]]` 형태로 로드됨